resources/language/en_us.ini

```diff
@@ -0,0 +1,1565 @@
+; this message gets displayed when the command executor hasn't the permissions to use the “/plot visit“ subcommand
+visit.permissionMessage = "§cYou don't have the permission to use this command."
+; this message gets displayed when the command executor isn't online (for example, if the command executor is the console)
+visit.senderNotOnline = "§cThis command must be run in-game."
+; This message gets displayed when the plots could not be loaded from the provider.
+; {%0} = error message
+visit.loadPlotsError = "§cThere was an error while loading plot§8: §7{%0}"
+; this message gets displayed when the command executor has no plots claimed
+visit.noArguments.noPlots = "§cYou don't own any plots."
+; this message gets displayed when there was an error during teleporting the command executor to the plot
+; {%0} = the world name of the plot
+; {%1} = the x coordinate of the plot
+; {%2} = the z coordinate of the plot
+visit.noArguments.teleportError = "§cThere was an error while teleporting you to plot (§6{%1}§c;§6{%2}§c)§8: §7{%0}"
+; this message gets displayed when the player was successfully teleported to the plot
+; {%0} = the world name of the plot
+; {%1} = the x coordinate of the plot
+; {%2} = the z coordinate of the plot
+visit.noArguments.success = "§7You were teleported to plot (§6{%1}§c;§6{%2}§c)§8: §7{%0}"
+; this message gets displayed when the command executor has no plots claimed
+visit.oneArgument.sender.noPlots = "§cYou don't own any plots."
+; this message gets displayed when the command executor has no plot with the provided number
+; {%0} = the provided plot number
+visit.oneArgument.sender.noPlot = "§cYou don't own plot §6{%0}§c."
+; this message gets displayed when there was an error during teleporting the command executor to the plot
+; {%0} = the provided plot number
+; {%1} = the world name of the plot
+; {%2} = the x coordinate of the plot
+; {%3} = the z coordinate of the plot
+visit.oneArgument.sender.teleportError = "§cThere was an error while teleporting you to plot (§6{%2}§c;§6{%3}§c)§8: §7{%0}"
+; this message gets displayed when the command executor was successfully teleported to the plot
+; {%0} = the provided plot number
+; {%1} = the world name of the plot
+; {%2} = the x coordinate of the plot
+; {%3} = the z coordinate of the plot
+visit.oneArgument.sender.success = "§7You were teleported to plot (§6{%2}§c;§6{%3}§c)§8: §7{%0}"
+; this message gets displayed when the provided player has no plots claimed
+; {%3} = the provided players name
+visit.oneArgument.player.noPlots = "§6{%0} §cdoesn't own any plots."
+; this message gets displayed when there was an error during teleporting the command executor to the plot
+; {%0} = the world name of the plot
+; {%1} = the x coordinate of the plot
+; {%2} = the z coordinate of the plot
+; {%3} = the provided players name
+visit.oneArgument.player.teleportError = "§cThere was an error while teleporting you to plot (§6{%1}§c;§6{%2}§c)§8: §7{%0}"
```

Comment:
The last parameter should be `{%3}` instead of `{%0}`
It might also be better to indicate that the plot is by a player, to better differentiate it from the alias case. So maybe `"§cThere was an error while teleporting you to the plot (§6{%1}§c;§6{%2}§c) of §6{%3}§c.`